tools/dockerfile/grpc_artifact_python_manylinux_x64/Dockerfile

```diff
@@ -34,6 +34,19 @@ FROM quay.io/pypa/manylinux1_x86_64
 # Update the package manager
 RUN yum update -y
 
+#############################################################
+# Update Git to allow cloning submodules with --reference arg
+RUN yum remove -y git
+RUN yum install -y curl-devel expat-devel gettext-devel openssl-devel zlib-devel gcc
+WORKDIR /usr/src
```

Comment:
I wouldn't use WORKDIR as that has lasting effect for the final docker image being built.

`RUN cd /usr/src  && wget FOO && tar  BAR` works just as well

Same below.